src/components/SingleReview.jsx

```diff
@@ -1,34 +1,74 @@
 import { useState, useEffect } from "react"
 import { useParams } from "react-router"
-import { getSingleReview } from "../api"
+import { getSingleReview, getComments} from "../api"
 
 const SingleReview = () => {
 
     const [isLoading, setIsLoading] = useState(true)
     const [review, setReview] = useState({})
+    const [comments, setComments] = useState([])
     const { review_id } = useParams()
     
     useEffect(() => {
+        setIsLoading(true)
         getSingleReview(review_id)
         .then((reviewFromApi) => {
             setReview(reviewFromApi)
             setIsLoading(false)
         })
+        getComments(review_id)
+        .then((commentsFromApi) => {
+            setComments(commentsFromApi)
```

Comment:
It would be great to have an isLoading for the comments section aswell as the article just incase these don't come back at the same time.